src/server/pfs/server/driver_v2.go

```diff
@@ -144,14 +145,17 @@ func (d *driverV2) finishCommitV2(txnCtx *txnenv.TransactionContext, commit *pfs
 func (d *driverV2) withFileSet(ctx context.Context, repo, commit string, f func(*fileset.FileSet) error) (retErr error) {
 	// TODO subFileSet will need to be incremented through postgres or etcd.
```

Comment:
It would probably be a good idea to refactor this into a helper function. The function would lock / defer unlock, then read and increment subfileset. It would return the subfileset value.